specification/provisioningservices/resource-manager/readme.md

```diff
@@ -29,15 +27,29 @@ openapi-type: arm
 tag: package-2017-08
 ```
 
+``` yaml
+openapi-type: arm
+tag: package-2017-11
+```
+
 ### Tag: package-2017-08
 
 These settings apply only when `--tag=package-2017-08` is specified on the command line.
```

Comment:
Description text is out of sync with yaml.

Reply:
resolved